src/PowerShellEditorServices/Hosting/HostStartupInfo.cs

```diff
@@ -0,0 +1,162 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation.Host;
+
+namespace Microsoft.PowerShell.EditorServices.Hosting
+{
+    /// <summary>
+    /// Contains details about the host as well as any other information
+    /// needed by Editor Services at startup time.
+    /// </summary>
+    public class HostStartupInfo
+    {
+        #region Constants
+
+        /// <summary>
+        /// The default host name for PowerShell Editor Services.  Used
+        /// if no host name is specified by the host application.
+        /// </summary>
+        public const string DefaultHostName = "PowerShell Editor Services Host";
+
+        /// <summary>
+        /// The default host ID for PowerShell Editor Services.  Used
+        /// for the host-specific profile path if no host ID is specified.
+        /// </summary>
+        public const string DefaultHostProfileId = "Microsoft.PowerShellEditorServices";
+
+        /// <summary>
+        /// The default host version for PowerShell Editor Services.  If
+        /// no version is specified by the host application, we use 0.0.0
+        /// to indicate a lack of version.
+        /// </summary>
+        public static readonly Version DefaultHostVersion = new Version("0.0.0");
```

Comment:
```suggestion
        public static readonly Version DefaultHostVersion = new Version(0, 0, 0);
```

Just to avoid the `Parse` call.